### ISSUE TITLE
cannot convert a simple network only contains convtranspose operator

### ISSUE THREAD

**Issue by videocanvas** [XiaoMi/mace#508]
****so simple this network, 
input (1,1,240,360)==> ConvTranspose(w:<1,1,3,3>,B<1>) ==>output(1,1,240,360) ,and yml file is as flowing** 
==================================================
library_name: sr120x120
target_abis: [armeabi-v7a, arm64-v8a]
model_graph_format: code
model_data_format: code
models:
  sr120x120:
    platform: onnx
    model_file_path: /mobile_android_test_1.onnx
    model_sha256_checksum: 859feba045d625c75cb8be0f192847eddf5891308ada41d48661872f6402795e
    subgraphs:
      - input_tensors: input
        output_tensors: output
        input_shapes: 1,240,360,1
        output_shapes: 1,240,360,1
        backend: caffe2
    runtime: cpu+gpu
    limit_opencl_kernel_time: 0
    nnlib_graph_mode: 0
    obfuscate: 0
    winograd : 0
==================================================**
**，and the error prompt info is as following ，
model.cc: In function 'void mace::{anonymous}::**CreateOperators_ex**(mace::NetDef*, int, int, int)':
mace/codegen/models/sr120x120/model.cc:112:3: ****error: 'CreateOperator1' is not a member of 'mace::sr120x120'****mace::sr120x120::CreateOperator1(net_def->add_op(),w,h,c*8);
   ^
mace/codegen/models/sr120x120/model.cc:113:3: error: 'CreateOperator2' is not a member of 'mace::sr120x120' mace::sr120x120::CreateOperator2(net_def->add_op(),w,h,c*12);**

**Comment by TonyMou** [comment]
Could you convert this model successfully and paste the printed graph info here?

**Comment by videocanvas** [comment]
the graph info is as following :
onnx model IR version:  3
constains ops domain:   version: 9
('node=', 'ConvTranspose', '35', [u'output'], [u'input', u'13', u'14'])
start transform ...
Transpose filters to OIHW/MIHW
Transpose output shapes: 35(Deconv2D)
update op with float data type
Add OpenCL informations
Sort by execution
Final ops:
35 (Deconv2D, index:0): [[1L, 240L, 360L, 1L]]
*********************************************
          Model sr120x120 converted          
*********************************************

**Comment by TonyMou** [comment]
Cannot locate where the problem is yet.
 Try the following settings in your .yml file :
     ```
     input_data_formats: NCHW
     output_data_formats: NCHW
```

Or provide your model here, we'll check it for you.

**Comment by videocanvas** [comment]
It doesn't work while modify the setting of .yml file as the following 
==========================
 input_data_formats: NCHW
 output_data_formats: NCHW
===========================
and my model is as the attach file
[mobile_android_test_1.zip](https://github.com/XiaoMi/mace/files/3499712/mobile_android_test_1.zip)

**Comment by TonyMou** [comment]
OK, We'll look into it.

**Comment by videocanvas** [comment]
maybe the problem exists in the file codegen/models/model.cc
why this function 
void CreateOperators_ex(NetDef *net_def,const int w,const int h,const int c) {
  MACE_LATENCY_LOGGER(1, "Create operators");

  net_def->mutable_op()->Reserve(1);
  **mace::sr1200x1200::CreateOperator0(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator1(net_def->add_op(),w,h,c*8);
  mace::sr1200x1200::CreateOperator2(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator3(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator4(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator5(net_def->add_op(),w,h,c*8);
  mace::sr1200x1200::CreateOperator6(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator7(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator8(net_def->add_op(),w,h,c*12);
  mace::sr1200x1200::CreateOperator9(net_def->add_op(),w,h,c*40);
  mace::sr1200x1200::CreateOperator10(net_def->add_op(),w*2,h*2,c*10);
  mace::sr1200x1200::CreateOperator11(net_def->add_op(),w*2,h*2,c);**
}
these operators are the operators of previous network transform, why does this tool still follow the
previous transform?

**Comment by TonyMou** [comment]
Already fixed by commit: 1fc9f7edc1492296ca847657f227e70272822887
